packages/lab/src/tabs-next/TabsNext.tsx

```diff
@@ -0,0 +1,74 @@
+import { Children, cloneElement, isValidElement, ReactNode } from "react";
+import { StackLayout, useControlled, useId } from "@salt-ds/core";
+import { TabstripNext, TabstripNextProps } from "./TabstripNext";
+import { TabNext } from "./TabNext";
+import { TabPanelNext } from "./TabPanelNext";
+import { TabElement } from "./TabsNextTypes";
+
+export type TabsNextProps = Omit<TabstripNextProps, "defaultActiveTabIndex"> & {
+  defaultSelectedTab?: string;
+};
+
+function isTabPanel(child: ReactNode | TabElement): child is TabElement {
+  return isValidElement(child) && child.type === TabPanelNext;
+}
+
+export const TabsNext = ({
+  children,
+  selectedTab: selectedTabProp,
+  defaultSelectedTab,
+  onSelectTab: onSelectTabProp,
+  ...props
+}: TabsNextProps) => {
+  const tabs = Children.toArray(children).filter(isTabPanel);
+  const [selectedTab, setSelectedTab] = useControlled({
+    controlled: selectedTabProp,
+    default: defaultSelectedTab ?? tabs[0]?.props.value,
+    name: "useTabs",
+    state: "selectedTab",
+  });
+
+  const uniqueId = useId();
```

Comment:
Should this honour id passed in?

Reply:
Yes, this makes sure consumer could apply an ID of choice in the application so they can write logic or test against it.
Thus why useId accepts a default.

Reply:
Sounds good